Makefile

```diff
@@ -16,6 +16,7 @@ PARSE_INTEGER_BENCH_CSV_FILE?=tfhe_rs_integer_benches.csv
 FAST_TESTS?=FALSE
 FAST_BENCH?=FALSE
 BENCH_OP_FLAVOR?=DEFAULT
+COVERAGE_EXCLUDED_FILES = tfhe/benches/*,apps/trivium/src/*,tfhe/examples/*,tasks/src/*
```

Comment:
I think it's fine to hardcode in the recipe, those are not supposed to change right ?

Reply:
It's because when I'll add the other layers I'll reuse this variable.

Reply:
not sure I follow 🤔

Reply:
It's to avoid copy/pasting since when I'll make `test_integer_cov`, `test_core_cov`, etc, I'll reuse this line of excluded files.

Reply:
ah yes makes sense then